.github/workflows/python-tests.yaml

```diff
@@ -80,7 +80,7 @@ jobs:
 
       fail-fast: true
 
-    timeout-minutes: 10
+    timeout-minutes: 15
```

Comment:
why?

Reply:
For some reason, the tests are running slower on 3.12, and I was just barely timing out due to new tests. I bumped the timeout, but I want to find out why tests are running slower on 3.12